UnitsNet/GeneratedCode/Quantities/Acceleration.g.cs

```diff
@@ -72,6 +72,39 @@ static Acceleration()
                     new UnitInfo<AccelerationUnit>(AccelerationUnit.StandardGravity, "StandardGravity", new BaseUnits(length: LengthUnit.Meter, time: DurationUnit.Second)),
                 },
                 BaseUnit, Zero, BaseDimensions, QuantityType.Acceleration);
+
+            // Register in default unit converter: BaseUnit -> AccelerationUnit
+            UnitConverter.Default.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.CentimeterPerSecondSquared, quantity => quantity.ToUnit(AccelerationUnit.CentimeterPerSecondSquared));
```

Comment:
Ok, interesting idea to break things up while not requiring the client to call some init method of sorts. It could help with both the issue and startup time.

I am wondering, can this result in conversions failing in some way? That they are not initialized yet, because the Acceleration static ctor just wasn't called yet in that particular call path?

Reply:
Aha, yes. If you call the following:
```csharp
var func = UnitConverter.Default.GetConversionFunction<Acceleration>( AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.KilometerPerSecondSquared );
```

It fails since it has not been registered yet, despite having the Acceleration type utilized. Let me see if I can fix this.

Reply:
Ok, I created a separate internal static RegisterDefaultConversions method in each of our quantities. I use reflection on the first call to the UnitConverter (as before) to register them all. This makes sure everything is initialized as-before, but without a giant method with all the types hard-coded into it.

Reply:
Nice 🐷💄